ims/company/password_and_2fa.md

```diff
@@ -0,0 +1,46 @@
+# Password and 2FA policy
+
+Strong password and appropriate use of 2FA are important aspects of device and software security. We all have passwords for multiple systems that are used for business related purposes and it is the responsibility of all users who have access to Made Tech's systems, including contractors, to take the appropriate steps to secure their passwords. 
+
+## Password Creation
+- Passwords must be sufficiently strong such as to make cracking or guessing difficult.
+- Where possible, passwords should be randomly generated
+- Passwords should be more than 12 characters and contain a mixture of upper case, lower case, numbers and special characters
+- You must not use the same password for Made Tech access as for non-Made Tech/personal access to services or devices.
+
+## Password Management
+We use password management for Made Tech services. The password manager is maintained by the IMS management group. Individual vaults are managed by the teams who use them.
+
+Your password to access your password manager should confirm to the password policy. Biometrics is also an acceptable method of accessing password managers.
+
+If you have reason to believe your password manager password has been compromised, report it immediately to the IMS manager or CSO. 
+
+## Obtaing Passwords via Social Engineering
+
+Cracking passwords or a brute force attack are not the only ways in which individuals with bad intentions can get into the Made Tech systems. Social engineering also makes us vulnerable. Attackers will use techniques to gain your trust, or take advantage of inattention or a desire to be helpful to obtain passwords. Keep in mind the following:
+
+- Made Tech passwords are shared on a need-to-know basis. All passwords are to be treated as sensitive, internal information belonging to Made Tech
+- Only share Made Tech passwords with persons / systems authorised by the system owner, and only share by secure means
+- Passwords must not be revealed over the phone
+- Do not write passwords down and store them anywhere in the office. 
+- Do not store passwords in a file on a computer system or mobile device (phone, tablet) without encryption.
+- Do not reveal a password on forms
+- Do not hint at the format of a password ("My hometown/name/etc)
+- Be sceptical of unusual requests
+- Be aware of potential phishing attacks
+- Consider who may be shoulder surfing
+- Hover your mouse over links in e-mail to ensure the web address makes sense
+- Verify the identity of a requester before sharing information
+- When in doubt, do not respond!
+
+It's always better to be safe than sorry. Always check with the IMS Manager or CSO is you think something is amiss.
+
+## 2FA
+Where a service provides the ability to authenticate via 2FA, 2FA must be used. 
+
+# Related Standards, Policies and Processes
+ - [AUP Policy](link)
+ - [Lost or Stolen policy](link)
+ - [How to report an incident](link)
+ - [Device Security](link)
+ - [BYOD Policy](link)
```

Comment:
Can we just use the full name here, rather than an acronym?

Reply:
BYOD was already in use on the old handbook. Well established precedent for the acronym 😉